Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -92,6 +93,22 @@ AddNewCardViewModelOutputs {
       self.addNewCardSuccess.mapConst(false),
       self.addNewCardFailure.mapConst(false)
     )
+
+    // Koala
+    self.viewWillAppearProperty.signal
+      .observeValues {
+        AppEnvironment.current.koala.trackViewedAddNewCard()
```

Comment:
Is there a reason for these 3 tracking events to be on a separate line? Should fit on 1 line..

```
self.viewWillAppearProperty.signal
      .observeValues { AppEnvironment.current.koala.trackViewedAddNewCard() }

    self.addNewCardSuccess
      .observeValues { _ in AppEnvironment.current.koala.trackSavedPaymentMethod() }

    self.addNewCardFailure
      .observeValues { _ in AppEnvironment.current.koala.trackFailedPaymentMethodCreation() }
```

Reply:
No reason :) Wil change it